resources/docs/livewire/editing-chirps.md

```diff
@@ -0,0 +1,517 @@
+            <svg xmlns="http://www.w3.org/2000/svg" class="h-6 w-6 text-gray-600 -scale-x-100" fill="none" viewBox="0 0 24 24" stroke="currentColor" stroke-width="2">
+                <path stroke-linecap="round" stroke-linejoin="round" d="M8 12h.01M12 12h.01M16 12h.01M21 12c0 4.418-4.03 8-9 8a9.863 9.863 0 01-4.255-.949L3 20l1.395-3.72C3.512 15.042 3 13.574 3 12c0-4.418 4.03-8 9-8s9 3.582 9 8z" />
+            </svg>
+            <div class="flex-1">
+                <div class="flex justify-between items-center">
+                    <div>
+                        <span class="text-gray-800">{{ $chirp->user->name }}</span>
+                        <small class="ml-2 text-sm text-gray-600">{{ $chirp->created_at->format('j M Y, g:i a') }}</small>
+                        @unless ($chirp->created_at->eq($chirp->updated_at))<!-- [tl! add:start] -->
+                            <small class="text-sm text-gray-600"> &middot; {{ __('edited') }}</small>
+                        @endunless<!-- [tl! add:end] -->
+                    </div>
+                    @if ($chirp->user->is(auth()->user()))<!-- [tl! add:start] -->
+                        <x-dropdown>
+                            <x-slot name="trigger">
+                                <button>
+                                    <svg xmlns="http://www.w3.org/2000/svg" class="h-4 w-4 text-gray-400" viewBox="0 0 20 20" fill="currentColor">
+                                        <path d="M6 10a2 2 0 11-4 0 2 2 0 014 0zM12 10a2 2 0 11-4 0 2 2 0 014 0zM16 12a2 2 0 100-4 2 2 0 000 4z" />
+                                    </svg>
+                                </button>
+                            </x-slot>
+                            <x-slot name="content">
+                                <x-dropdown-link wire:click="edit({{ $chirp->id }})">
+                                    {{ __('Edit') }}
+                                </x-dropdown-link>
+                            </x-slot>
+                        </x-dropdown>
+                    @endif<!-- [tl! add:end] -->
+                </div>
+                <p class="mt-4 text-lg text-gray-900">{{ $chirp->message }}</p><!-- [tl! remove] -->
+                @if ($chirp->is($editing)) <!-- [tl! add:start] -->
+                    <livewire:chirps.edit :chirp="$chirp" :key="$chirp->id" />
+                @else
+                    <p class="mt-4 text-lg text-gray-900">{{ $chirp->message }}</p>
+                @endif <!-- [tl! add:end] -->
+            </div>
+        </div>
+    @endforeach
+</div>
+```
+
+## Creating the edit form
+
+Let's now create the `chirps.edit` Livewire component:
+
+```shell tab=Class
+php artisan make:volt chirps/edit --class
+```
+
+```shell tab=Functional
+php artisan make:volt chirps/edit
+```
+
+This will create a new Livewire component at the `app/resources/views/livewire/chirps/edit.blade.php` path. Let's update the Livewire component contents to display a form for editing a Chirp.
```

Comment:
```suggestion
This will create a new Livewire component at the `resources/views/livewire/chirps/edit.blade.php` path. Let's update the Livewire component contents to display a form for editing a Chirp.
```

Reply:
Fixed.